authentication/views.py

```diff
@@ -333,6 +333,33 @@ def delete(self, request):
         return Response("Company Deleted Successfully", status.HTTP_200_OK)
 
 
+class ClientListView(generics.GenericAPIView):
+    """
+    Return a list of registered clients
+    """
+    serializer_class = ClientSerializer
+    renderer_classes = (ClientJSONRenderer,)
+    permission_classes = (IsAuthenticated,)
+
+    def get_queryset(self):
```

Comment:
Hey @Rhytah , great work. Kindly add a docstring here explaining what this queryset is returning

Reply:
@Genza999 done